CHANGELOG.rst

```diff
@@ -1,6 +1,21 @@
 Core Framework change log
 =========================
 
+1.0.2
```

Comment:
in fwk_xvf we used the following style:

```
- CHANGED: lib_xshf updated from v1.9.0 to v2.0.0
- CHANGED: xmos_cmake_toolchain updated from untagged commit e577fbc to v1.0.0
```
I find useful to know the previous version, especially if this was not listed in older information.

Reply:
Yes I think that makes sense. I will update.